sagemaker-python-sdk/tensorflow_script_mode_using_shell_commands/tensorflow_script_mode_using_shell_commands.ipynb

```diff
@@ -0,0 +1,303 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# TensorFlow Script Mode - Using Shell commands\n",
+    "\n",
+    "Starting by TensorFlow's framework version 1.11, you can shell commands as\n",
+    "your training entry point. Shell scripts are useful for many use cases including:\n",
+    "\n",
+    "- Invoke Python scripts with specific parameters\n",
+    "- Install requirements before training\n",
+    "- Configure framework dependencies\n",
+    "- Train using different computer languages\n",
+    "\n",
+    "For this example, you use [a Keras implementation of the Deep Dream algorithm](https://github.com/keras-team/keras/blob/2.2.4/examples/deep_dream.py). You can use the same technique for other scripts or repositories. For example, [TensorFlow Model Zoo](https://github.com/tensorflow/models) and [TensorFlow benchmark scripts](https://github.com/tensorflow/benchmarks/tree/master/scripts/tf_cnn_benchmarks)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting the image for training\n",
+    "For training data, let's download an open sourced image:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "data_dir = os.path.join(os.getcwd(), 'training')\n",
+    "\n",
+    "os.makedirs(data_dir, exist_ok=True)\n",
+    "data_dir"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget -O training/resting-dragon.jpg https://s3.amazonaws.com/wyspstore/1360008n.jpg"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.display import Image\n",
+    "Image(filename='training/resting-dragon.jpg') "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download the training script\n",
+    "\n",
+    "Let's start downloading the training script:"
```

Comment:
I'd change this to either "Let's start by downloading the training script. This script comes from blah blah blah and does blah blah blah." or "Let's download a script from blah blah blah that does blah blah blah."